app/src/main/java/com/github/orkest/View/auth/AuthActivity.kt

```diff
@@ -0,0 +1,50 @@
+package com.github.orkest.View.auth
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.rememberNavController
+import com.github.orkest.View.theme.OrkestTheme
+import androidx.navigation.compose.composable
+
+class AuthActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val viewModel = AuthViewModel()
+        setContent {
+            OrkestTheme {
+                // A surface container using the 'background' color from the theme
+                Surface(
+                    modifier = Modifier.fillMaxSize(),
+                    color = MaterialTheme.colorScheme.background
+                ) {
+                    authMain(viewModel)
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun authMain(viewModel: AuthViewModel) {
+    val navController = rememberNavController()
+
+    NavHost(navController = navController, startDestination = "signup") {
+        composable("signup") { SignUpForm(navController = navController,viewModel) }
+    }
+}
+
```

Comment:
Why do we need to use navigation for this activity ?

Reply:
I thought it would be a nice way to navigate between the authentication pages (signing up, signing in, going directly to the main activity of the app if the user is signed in..) but I think we'll have to discuss it more when we will have all the other parts of the app to put in context